src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java

```diff
@@ -69,11 +72,40 @@ private String getDeviceIp(Device device) {
   }
 
   private boolean connectDevice(String deviceIp) {
-    String enableTCPCommand = getCommand("tcpip 5555");
-    commandLine.executeCommand(enableTCPCommand);
+    enableTCPCommand();
     String connectDeviceCommand = getCommand("connect " + deviceIp);
-    String connectOutput = commandLine.executeCommand(connectDeviceCommand);
-    return connectOutput.contains("connected");
+    return commandLine.executeCommand(connectDeviceCommand).contains("connected");
+  }
+
+  /**
+   * Disconnect connected device by IP.
+   * If device is disconnected with success, we should get an empty command result.
+   * @param deviceIp
+   * @return
+   */
```

Comment:
We are not using javadoc in this project. We should remove this, the method name is readable enough :)